Disaster Recovery/SafeHaven 5 General/SafeHaven5.0.0-Release-Notes.md

```diff
@@ -39,18 +39,9 @@ This is the first major release in the SafeHaven-5 product line and there is no
 
 **NOTE**: Upgrade from SafeHaven-4 to SafeHaven-5 will be a fresh new installation.
 
-### Download Links
-* [SafeHavenAppliance-5.0.0.ova](https://download.safehaven.ctl.io/SH-5.0.0/SafeHavenAppliance-5.0.0.ova): OVA to be imported to private VMware vSphere or vCloud Director envrionment as templates for SRN/CMS appliances
-* [Ubuntu-16.vhdx](https://download.safehaven.ctl.io/SH-5.0.0/Ubuntu-16.vhdx): Hyper-V image for SRN/CMS appliances in a private Hyper-V envrionment
-* [Debian Package for CMS/SRN](https://download.safehaven.ctl.io/SH-5.0.0/safehaven-5.0.0.deb): to be used in the cluster installer
-* [GUI Package](https://download.safehaven.ctl.io/SH-5.0.0/SafeHavenConsole-5.0.0.zip): zip file containing the JRE, runs on any Windows platform
-  * [SafeHavenClient.5.0.0_win32.jar](https://download.safehaven.ctl.io/SH-5.0.0/SafeHavenClient.5.0.0_win32.jar): the Java application that requires an existing 32bit JRE-8 to run on a Windows OS
-* Windows Downloads:
-  * [Driver Installer](https://download.safehaven.ctl.io/SH-5.0.0/safehaven_windows_driver-5.0.0.exe)
-  * [MakeStub.exe](https://download.safehaven.ctl.io/SH-5.0.0/MakeStub-5.0.0.exe)
-* [Linux SafeHaven Onboarding Scripts Downloads](https://download.safehaven.ctl.io/SH-5.0.0/safehaven_linux_onboarding_scripts-5.0.0.tar.gz)
-
-Please note that you can check the md5 checksum against the file named [MD5SUMS](https://download.safehaven.ctl.io/SH-5.0.0/MD5SUMS).
+### SafeHaven Most Recent Releases
+Please take a look at the link below to download the latest available release  
+https://www.ctl.io/knowledge-base/disaster-recovery/overview/most-recent-safehaven-release-updates/
```

Comment:
change to `[Most recent updates](../../Disaster Recovery/Overview/most-recent-safehaven-release-updates.md)`